gamestonk_terminal/portfolio/portfolio_helper.py

```diff
@@ -60,3 +60,23 @@ def clean_name(name: str) -> str:
         A cleaned value
     """
     return name.replace("beta_", "").upper()
+
+
+def get_fraction(n: float, d: float) -> str:
+    """Turn two numbers into a fraction
+
+    Parameters
+    ----------
+    n : float
```

Comment:
I'd recommend more descriptive names.

numerator instead of n and denominator instead of d.